Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -386,7 +387,10 @@ export class dotNetExe {
     }
 
     private static getModifiedOutputForProjectFile(outputBase: string, projectFile: string): string {
-        return path.join(outputBase, path.basename(path.dirname(projectFile)));
+        let projectFileName = path.basename(projectFile);
+        let indexOfFileSeperator = projectFileName.indexOf('.');
+        projectFileName = projectFileName.slice(0, indexOfFileSeperator ? indexOfFileSeperator : projectFileName.length);
```

Comment:
Can filename not have '.'?

Reply:
They should have '.' always when its a csproj file. But at least on windows, file without extensions can exist, hence handled that.

Reply:
What I meant was what if there are multiple '.' in the file name, for example "a.b.extension". Is this possible?

Reply:
yeah that's right.... 
But removed the logic of file name now. Nice catch